### PR TITLE
K_Query folder added with separate readme.

### PR DESCRIPTION
**Pull request by uhini0201**
## Related Issue
- #60 

## Proposed Changes
- Added the K_query problem
- Created a folder named fenwick tree which has its own readme and also added the problem.
## Additional Info
- Each folder has its individual readme.

## Checklist

- [X] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [X] 🎀 I have updated the README accordingly.
- [X] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [X] ✨ I have added tests to cover my changes.
- [X] 🚩 All new and existing tests passed.
- [X] 🌟 ed the repo

I have added the K_query problem in cpp in details.

This is a very well known algorithm and quite efficient and interesting the world of CP.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | 
| ----------- | 
| Output1  | 
| output1  |

### CODE REVIEW

**Comment by uhini0201** [comment]
@miraj0507 @Lakhankumawat Please check the let me know if anything else needed.

**Comment by Lakhankumawat** [comment]
I see there are zipped files and png files which are not allowed

**Comment by Lakhankumawat** [comment]
you must have only one c++ file with a README in which everything related to what you have contributed 



You need to follow particular guidelines for your README , please follow
these guidelines [click here](https://github.com/Lakhankumawat/LearnCPP#to-add-programs-using-stl-follow-same-approach-as--algorithms) 
also you can see how your README should look like [see here](https://github.com/Lakhankumawat/LearnCPP/blob/main/S-SortingAlgorithms/README.md) 
for more please go through these [here](https://youtu.be/7UgC3HDETRA)

**Comment by Lakhankumawat** [comment]
Change your Code Directory to this ` LearnCPP/T-Tree/F-FenwickTree/KQuery.cpp and README.md`

**Comment by Lakhankumawat** [comment]
Change the title of your Pull Request and make it related to your issue or Pr type

**Comment by uhini0201** [comment]
Ok I am removing the zip files and the png files are for readme. @Lakhankumawat

**Comment by uhini0201** [comment]
@Lakhankumawat Please check now.

**Comment by uhini0201** [comment]
@Lakhankumawat please update if anything else is needed.

**Comment by uhini0201** [comment]
@Lakhankumawat I guess now this pr is ready to merge. If anything else is needed, please let me know.

**Review by uhini0201 — COMMENTED** [review]
I have reviewed them

**Comment by uhini0201** [comment]
@miraj0507 can you please go through the changes???

**Comment by uhini0201** [comment]
@Lakhankumawat @miraj0507 Please check my PR #561 if any other change is required.

**Comment by uhini0201** [comment]
@Lakhankumawat

**Comment by uhini0201** [comment]
@Lakhankumawat I am waiting for you to update with what else is needed to be added yet. Please tell me the changes??

**Comment by uhini0201** [comment]
@Lakhankumawat @miraj0507 I am still waiting for your reply.

File: F-FenwickTree/K_Query.cpp [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#include<stdio.h>
+#include<bits/stdc++.h>
+#define lli long long int
+#include<utility>
```
Comment (Lakhankumawat) [inline F-FenwickTree/K_Query.cpp:4]: it will be great if you add comments to your code

File: F-FenwickTree/Readme.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
+
+# Table Of Content
+
```
Comment (Lakhankumawat) [inline F-FenwickTree/Readme.md:3]: Your README looks good ✨

**Review by Lakhankumawat — CHANGES_REQUESTED** [review]

**Comment by Lakhankumawat** [comment]
And sorry for the delay @miraj0507  is inactive as mentor , i am admin and i only see PR's at the end to merge them reviewing PR's is not my work , hope you understand , but now if you do the mentioned changes i will merge your pr

**Comment by miraj0507** [comment]
Don't worry will review your PR soon about Fenwick tree

File: F-FenwickTree/Readme.md (hunk 2 above)
Comment (uhini0201) [reply F-FenwickTree/Readme.md:3]: Thanks a lot, i was not sure of the way i did it.

File: F-FenwickTree/K_Query.cpp [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#include<stdio.h>
```
[the remark this answers is not in the document]
Comment (uhini0201) [reply F-FenwickTree/K_Query.cpp]: Done Changed the directory to the specified spot. @Lakhankumawat

File: F-FenwickTree/K_Query.cpp (hunk 1 above)
Comment (uhini0201) [reply F-FenwickTree/K_Query.cpp]: The comments are added. @Lakhankumawat @miraj0507

**Review by Lakhankumawat — COMMENTED** [review]
LGTM 🌟

**Comment by Lakhankumawat** [comment]
@all-contributors please add @uhini0201 code ,

**Pull request merged**